src/internal/observe/observe.go

```diff
@@ -14,15 +14,16 @@ import (
 )
 
 const (
-	noProgressBarsFN = "no-progress-bars"
-	progressBarWidth = 32
+	noProgressBarsFN   = "no-progress-bars"
+	saveProgressBarsFN = "save-progress-bars"
```

Comment:
```suggestion
	saveProgressBarsFN = "retain-progress"
```
I'd prefer the shorter switch

Reply:
changed